readthedocs/projects/tasks/utils.py

```diff
@@ -99,7 +99,7 @@ def finish_inactive_builds():
     # Set time as maximum celery task time limit + 5m
     time_limit = 7200 + 300
     delta = datetime.timedelta(seconds=time_limit)
-    query = ~Q(state__in=BUILD_FINAL_STATES) & Q(date__lte=timezone.now() - delta)
+    query = ~Q(state__in=BUILD_FINAL_STATES) & Q(date__lt=timezone.now() - delta)
```

Comment:
Looks like this shouldn't be changed?

Reply:
I think it changed it because it makes the query faster. There is no chance to have an equal on an exact datetime. So, I'm +1 on this change

Reply:
I don't see any mention of this type of fixes in https://github.com/adamchainz/django-upgrade#fixers, it might be from some changes you had locally?

Reply:
Maybe something weird happens, yeah. I'll recheck.

Reply:
Done! I've reverted this change from here. I'll open a new PR for this.